ansible/roles/elasticsearch/templates/config/elasticsearch.yml.j2

```diff
@@ -9,11 +9,6 @@ node.name: "{{ elasticsearch_node_name }}"
 
 transport.tcp.port: {{ elasticsearch_tcp_port }}
 
-{% if elasticsearch_bind_to_localhost %}
-# don't expose ES to public
-network.bind_host: localhost
```

Comment:
this was actually just wrong. `host` is set to an IP address not localhost and thus it made it impossible to even access elastic search. the reason it didn't surface sooner is that no one restarted ES when the change was made